src/Sylius/Bundle/AttributeBundle/Resources/views/attributeChoice.html.twig

```diff
@@ -10,9 +10,9 @@
                     <div class="list-group row">
```

Comment:
Hi @Zales0123,

I'm just updating our backend for an entity to use this attribute UI - looking good! 

The only issue I'm having is that this line here is coupled to ProductAttributes (and I need to use it for a different model), as you can see from the routing:

```
sylius_backend_render_attribute_forms:
    path: /attribute-forms
    methods: [GET]
    defaults:
        _controller: sylius.controller.product_attribute:renderAttributeValueFormsAction
```

Is this something that you can look at?